src/components/grouped-character/grouped-character.js

```diff
@@ -139,12 +139,12 @@ const GroupedCharacter = Input(InputLabel(InputValidation(
 
   isValidKeypress = (ev) => {
     return (
-      !Events.isNumberKey(ev) &&
-      !Events.isAlphabetKey(ev) &&
-      !Events.isTabKey(ev) &&
-      !Events.isDeleteKey(ev) &&
-      !Events.isBackspaceKey(ev) &&
-      !Events.isNavigationKey(ev)
+      !Events.isNumberKey(ev)
```

Comment:
I prefer the previous

Reply:
I'm indifferent - will go with whatever people prefer. See https://eslint.org/docs/rules/operator-linebreak

Reply:
One advantage to having them down the left means you can look straight down the left edge to see the operators.

Reply:
I tried switching it and comparing the diff, it is actually easier to understand whats happening with the operator on the left - otherwise it gets lost at the end of the line.